703-Add db_connection param to POST Workspace Layers (#734)
LayerManager/layman: doc/rest.md

Publish vector or raster data file as new layer of WMS, in case of vector also new feature type of WFS.

Processing chain consists of few steps:

> - save eventual files to workspace directory within Layman data directory

Reviewer: ```suggestion
- save files (if sent) to workspace directory within Layman data directory
```

- save basic information (name, title, access_rights) into PostgreSQL

> - for vector layers import eventual vector file to PostgreSQL database as new table into workspace schema

Reviewer: ```suggestion
- for vector layers import vector file (if sent) to PostgreSQL database as new table into workspace schema
```

Content-Type: `multipart/form-data`, `application/x-www-form-urlencoded`

Body parameters:

> - *file*, file(s) or file name(s)

Reviewer: ```suggestion
- *file*, file(s) or file name(s)
  - exactly one of `file` or `db_connection` must be set
```

   - if published file has empty bounding box (i.e. no features), its bounding box on WMS/WFS endpoint is set to the whole World
   - attribute names are [laundered](https://gdal.org/drivers/vector/pg.html#layer-creation-options) to be safely stored in DB
   - if QML style is used in this request, it must list all attributes contained in given data file
- *db_connection*, string

>   - is not allowed in combination with *file* parameter

Reviewer: ```suggestion
  - exactly one of `file` or `db_connection` must be set
```
